Sources/App/Views/PackageController/MaintainerInfo/MaintainerInfoIndex+Model.swift

```diff
@@ -78,3 +78,45 @@ extension MaintainerInfoIndex {
         }
     }
 }
+
+extension Score.ScoreDetails.ScoreCategory {
+    var title: String {
+        switch self {
+        case .archive: return "Archived"
+        case .license: return "License"
+        case .releases: return "Releases"
+        case .stars: return "Stars"
+        case .dependencies: return "Dependencies"
+        case .maintenance: return "Last Activity"
+        case .documentation: return "Documentation"
+        case .readme: return "README"
+        case .contributors: return "Contributors"
+        case .tests: return "Tests"
+        }
+    }
+    
+    func description(candidate: Score.Input) -> String {
+        switch self {
+        case .archive:
+            "Repository is \(candidate.isArchived ? "" : "not") archived."
+        case .license:
+            "OSI-compatible license \(candidate.licenseKind == .compatibleWithAppStore ? "" : "not") which is compatible with the App Store."
+        case .releases:
+            "Has \(pluralizedCount: candidate.releaseCount, singular: "release")."
+        case .stars:
+            "Has \(pluralizedCount: candidate.likeCount, singular: "star")."
+        case .dependencies:
+            "\(candidate.numberOfDependencies ?? 0 < 1 ? "Has no dependencies." : "Depends on \(pluralizedCount: candidate.numberOfDependencies ?? 0, singular: "package", plural: "packages").")"
+        case .maintenance:
+            if candidate.lastActivityAt != nil { "The last maintenance activity was \(candidate.lastActivityAt?.relative)." } else { "No data available."}
```

Comment:
@daveverwer Btw, I updated this text to show "No data available" if the lastActivityAt date is nil. Let me know if you prefer a different way of wording this.

Reply:
I made a bit of a change to how we handle this. Instead of having an optional, I made `lastActivityAt` non-optional so we don't need to deal with `nil` values here. I also added some logic to update the text say "No recent maintenance activity" if there has been nothing in the last two years, and set a very old date for the non-optional `lastActivityAt`.

It's all in one commit 575ea8e64cc8c86f8bb26b0bb96dda8274f7fde5 if you'd like to take a look.

Reply:
great, thank you for the updates! looks a lot better.